resources/aws/routetable.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+type RouteTable struct {
+	Name  string
+	VpcID string
+	id    string
```

Comment:
I want to keep that field private. The only places that modifiy the ID, should be `CreateOrFail` and `CreateIfNotExists` methods, then we should be able to only get it through the interface method.